src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -361,6 +361,61 @@ function rhs_parabolic!(du_ode, u_ode, semi::SemidiscretizationHyperbolicParabol
     return nothing
 end
 
+"""
+    linear_structure(semi::SemidiscretizationHyperbolicParabolic;
+                     t0=zero(real(semi)))
+
+Wraps the right-hand side operator of the hyperbolic-parabolic semidiscretization `semi`
+at time `t0` as an affine-linear operator given by a linear operator `A`
+and a vector `b`:
```

Comment:
"Wraps" somehow sounds to me as if it would be another _equivalent_ formulation of the right-hand side, but in the case of nonlinear equations it's a linearization, right? Can we make this more explicit (same holds for the corresponding method for (purely) hyperbolic equations)?

Reply:
Hm yeah I guess the term `wrap` was chosen to somehow reflect that this is a different way of writing the `rhs!` 

Having a quick look at https://github.com/search?q=repo%3Atrixi-framework%2FTrixi.jl%20linear_structure&type=code also reveals that we do not execute this on a nonlinear equation. Not sure if this works at all 🤔

Reply:
Well, if it doesn't work, it should be made even clearer in the docstring.

Reply:
It does not crash, but gives different results:

<img width="737" height="600" alt="Screenshot 2025-10-18 215213" src="https://github.com/user-attachments/assets/99effe30-b2ba-4027-b5ad-e124e04143a8" />